tauri/tauri.conf.json

```diff
@@ -20,6 +20,13 @@
       "category": "Entertainment",
       "shortDescription": "",
       "longDescription": "",
+      "icon": [
```

Comment:
This section is required for me on win 10. It refused to `tauri build` , `tauri dev` worked

Reply:
@endrl Check our [CI pipelines](https://github.com/jellyfin/jellyfin-vue/blob/master/.github/workflows/quality_checks.yml#L121) for windows, as I said before you need to generate icons first

Reply:
Yes, but no. Thats my output. How can CI pass this tauri check. If i delete `icons` folder `tauri dev` stops working too. No idea why ci is green
```
   Compiling jellyfin-vue-tauri v0.0.0 (C:\Users\usr\jellyfinvue\jellyfin-vue\tauri)
    Finished release [optimized] target(s) in 20.66s
thread '<unnamed>' panicked at 'the bundle config must have a `.ico` icon', src\interface\rust.rs:977:8
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
@endrl But it's that after `npm run tauri:icon -w tauri`?

Reply:
yes, icons are build. I deleted them now and run again `tauri build`
```´
 Compiling jellyfin-vue-tauri v0.0.0 (C:\Users\usr\jellyfinvue\jellyfin-vue\tauri)
error: proc macro panicked
 --> src\main.rs:8:10
  |
8 |     .run(tauri::generate_context!())
  |          ^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: failed to read icon C:\Users\usr\jellyfinvue\jellyfin-vue\tauri\icons/icon.png:
```

The build context message is clear. It makes no sense that ci works.

Reply:
@endrl But don't delete them!
What we don't want is to have them commited to source (in the same manner as we have node_modules ignored) but ofc you need to have them in the local filesystem!